Tutorials/Sprite-Related-Tutorials/Getting Started With Sprites in Splashkit Tutorial - C#/Getting Started With Sprites in Splashkit Tutorial - CSharp.mdx

```diff
@@ -195,13 +314,26 @@ reading on ahead. What specific process are we missing that would cause _every_
 
 Truth is it's just a single line, if we add in
 
-```
+<Tabs>
+  <TabItem label="C#">
+```csharp
   spriteWindow.Clear(Color.White);
 ```
+  </TabItem>
+  <TabItem label="C++">
+```c++
+clear_window(window wind, color clr)
```

Comment:
This should be something like `clear_window(spriteWindow, Color.White);`. I'm not sure if the colours are defined so you might need to pass in RGB values to get the white.